ui/src/app/views/workspace/component.html

```diff
@@ -166,8 +166,7 @@ <h6>
         </div>
       </div>
     </div>
-    <a href="https://console.cloud.google.com/storage/browser/{{workspace.googleBucketName}}?authuser={{username}}"
-        target="_blank" rel="noopener noreferrer">
+    <a rel="noopener noreferrer" (click)="googleBucketModal = true">
       Google Bucket
```

Comment:
Do you still need the `rel` attribute here? Also, semantically, this probably shouldn't be an anchor tag anymore. Maybe a button link?